decidim-core/app/views/decidim/errors/internal_server_error.html.erb

```diff
@@ -1,10 +1,32 @@
 <div class="wrapper">
   <div class="row">
     <div class="large-8 large-centered">
-    <div class="static__message">
-      <h1 class="heading1 page-title"><%= t(".title") %></h1>
-      <p><%= t(".try_later") %></p>
+      <div class="static__message">
+        <h1 class="heading1 page-title"><%= t(".title") %></h1>
+        <p><%= t(".try_later") %></p>
+        <div class="table-scroll">
+          <table class="table-list">
+          <% @info_hash.map do |key, val| %>
+            <tr>
+              <th><%= t(".#{key}") %></th>
+              <td><%= val %></td>
+            </tr>
+          <% end %>
+          </table>
+        </div>
+        <div class="input-group">
+          <textarea id="referenceData" class="hide"><%= @plain_info %></textarea>
+          <div class="input-group-button">
```

Comment:
I missed this on the previous review but I just noticed during the final review that the "copy" button layout is incorrect:
![Copy to clipboard](https://user-images.githubusercontent.com/864340/188128061-508d2b21-d771-43ae-ad6b-f3333bc38ca2.png)

The button is "cut" on the left side, i.e. it doesn't have the border radius on the left side. This is what the primary button normally looks like e.g. in the proposal cards:
![Proposal card example](https://user-images.githubusercontent.com/864340/188128223-5cd62ed7-0e8c-4bbf-b3cb-8332f954b57c.png)